core/src/main/java/org/infinispan/util/logging/events/EventLogSerializer.java

```diff
@@ -0,0 +1,39 @@
+package org.infinispan.util.logging.events;
+
+import java.util.Optional;
+
+import org.infinispan.commons.configuration.io.ConfigurationWriter;
+import org.infinispan.configuration.serializing.ConfigurationSerializer;
+
+/**
+ * @since 14.0
+ */
+public class EventLogSerializer implements ConfigurationSerializer<EventLog> {
+
+   @Override
+   public void serialize(ConfigurationWriter writer, EventLog event) {
+      writer.writeStartElement("log");
+      writer.writeAttribute("instant", event.getWhen().toString());
+      writer.writeAttribute("category", event.getCategory().name());
+
+      writer.writeStartElement("content");
+      writer.writeAttribute("level", event.getLevel().name());
+      writer.writeAttribute("message", event.getMessage());
+      writer.writeEndElement();
+
+      writer.writeStartElement("additional");
```

Comment:
Do we need the `additional` element or should we just include this as part of the `content` element? The `EventLog#getDetail` field strikes me as something that is definitely part of the "content". The other fields could be classed as meta data.

Reply:
I've added the `additional` so we could make a distinction of some kind between what was the message and what was just meta data. But I defined these arbitrarily, I am going to update it.